### DISCUSSION TITLE
Possible to install 2.4 on Oracle Linux 9?

### DISCUSSION THREAD

**Discussion by MMStandley** [Security-Onion-Solutions/securityonion#10854] [2.4] [answered]
After seeing the post about SO moving to Oracle 9, I decided to install it and then put 2.4 on it. 
However, I get the info error: We do not support the operating system you are trying to use. It looks like it's still checking for Rocky 9. 
Is there a way around this or do I just need to wait for an update to 2.4 or what? 

Thanks, have a fabulous day!

**Comment by TOoSmOotH** [comment, marked as answer]
We will have it supported in RC1 which is the next release.

**Reply by MMStandley** [reply]
Thanks sir! Have a great day!
